product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -16,65 +16,78 @@ Prior to creating your cluster, make sure you have enough resources. Without eno
 
 1.  Sign in to the [BigAnimal](https://portal.biganimal.com) portal.
 
-2.  Select **Create New Cluster** in the top right of the **Overview** page or **Clusters** page.
-3.  On the **Create Cluster** page, specify the cluster settings on the following tabs:
-    -   [**Cluster Settings**](#cluster-settings-tab)
+1.  Select **Create New Cluster** in the top right of the **Overview** page or **Clusters** page.
+1.  On the **Create Cluster** page, specify the cluster settings on the following tabs:
+    -   [**Cluster Type**](#cluster-type-tab)
 
+    -   [**Cluster Settings**](#cluster-settings-tab)
```

Comment:
the tabs should be cluster info, cluster settings and db configuration

Reply:
fixed